types/domexception/index.d.ts

```diff
@@ -1,7 +1,69 @@
-// Type definitions for domexception 2.0
+// Type definitions for domexception 4.0
 // Project: https://github.com/jsdom/domexception#readme
 // Definitions by: ExE Boss <https://github.com/ExE-Boss>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
+/** An abnormal event (called an exception) which occurs as a result of calling a method or accessing a property of a web API. */
+interface DOMException extends Error {
```

Comment:
Note that this definition won’t be merged with `globalThis.DOMException`, because this file is a module, and therefore this definition shadows `globalThis.DOMException`.